adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -275,7 +275,17 @@ private AuthenticationResult tryAcquireTokenSilent(final AuthenticationRequest a
 
         if (shouldTrySilentFlow(authenticationRequest)) {
             Logger.v(TAG, "Try to acquire token silently, return valid AT or use RT in the cache.");
-            authenticationResult = acquireTokenSilentFlow(authenticationRequest);
+            try {
```

Comment:
You're putting it in the wrong place. I mentioned to you on slack and said there is already a fix checked in for this, but it went to release branch directly. 

Here is the commit for it https://github.com/AzureAD/azure-activedirectory-library-for-android/commit/e6f34a208260a3af6f1fa13027e8148153998cb1